src/__tests__/country.test.js

```diff
@@ -0,0 +1,19 @@
+import renderer from 'react-test-renderer';
+import { MemoryRouter } from 'react-router-dom';
+import Country from '../components/country';
+
+describe('Country', () => {
```

Comment:
[OPTIONAL] You added a unit test for this component as the reviewer requested, but I highly recommend you add more tests. Testing is a must for building robust software. 👍 
Kindly check this articles to learn more about testing React and Redux: 
https://www.willowtreeapps.com/craft/best-practices-for-unit-testing-with-a-react-redux-approach
https://medium.com/expedia-group-tech/integration-testing-in-react-21f92a55a894